### PR TITLE
Docs-feedback fixes

### PR DESCRIPTION
**Pull request by george-hoffer**
See https://mulesoft.slack.com/archives/C02HGAD2E/p1545335430082500

### CODE REVIEW

File: modules/ROOT/pages/mule-oauth-provider-landing-page.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ In addition to Mule OAuth 2.0 Provider, Mulesoft offers out-of-the-box integrati
 
 *Notes:*
 
-*  Mule OAuth 2.0 Provider is not an actual OAuth provider and is not meant to take the place of an actual identity provider.
+*  Mule OAuth 2.0 Provider is not an actual OAuth provider and is not meant to take the place of an actual identity provider. You can still use OAuth 2.0 Provider in production but not as your *only* OAuth Provider. You still need some sort of store or IDP in the backend while the provider can be how your application integrates with the Mule platform.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/mule-oauth-provider-landing-page.adoc]: Revise for clarity, concision? Such as:

You can use OAuth 2.0 Provider in production to integrate directly with Anypoint Platform. However, you also need an IdP or store to manage (back-end transactions? Authorization into your production environment?). Why would anyone do that? Should we just go with the original comments?

**Review by Mulesoft-Mysti — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/mule-oauth-provider-landing-page.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mule-oauth-provider-landing-page.adoc]: I don't think Mule platform and Anypoint Platform are the same. Mule platform is more of a catch-all for Mule on prem and cloud, and standalone or via Anypoint Platform.
How about:
OAuth 2.0 Provider should not be the only OAuth provider. Use this product in conjunction with an identity provider to handle authorization in a Mule application environment.

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/mule-oauth-provider-landing-page.adoc]: There is no such product as Mule platform so we can't say that. You can say Mule to mean all those things. Your rewrite feels less specific. Do apps integrate with Mule? I thought they used Mule to integrate with each other.  So are we saying you can't use it between apps, but can to manage identity between Mule and an app that you're integrating with Mule (but not app-to-app?).

Comment (evangelina) [reply modules/ROOT/pages/mule-oauth-provider-landing-page.adoc]: @chiragnarangmulesoft , could you please review the statement around OAuth 2.0 provider in production and confirm?

**Comment by glenn-rodgers-sf** [comment]
Closing as PR is no longer relevant.

**Pull request closed without merging**